Fail to reference 1.0.0 DL4J beta jar in Apache Tika project

See: https://github.com/apache/tika/pull/241#issuecomment-403234614 for more explanation. Should have worked by referencing the snapshots repo but it didn't

thanks @saudet I tried this and also tried building DL4J locally on my Mac OS X. It currently:

1. Hangs every time downloading some library. Will post in a sec when it hangs again.
2. The first time it built but I forgot to have cmake installed so it died. FML.

So I can't even build on my Mac. Where should I put the above command? In Tika? I'll try it there.

I tried your workaround, I got:

```
Downloaded: https://oss.sonatype.org/content/repositories/snapshots/org/nd4j/nd4j-api/1.0.0-SNAPSHOT/nd4j-api-1.0.0-20180709.040131-4516.jar (5.4 MB at 2.7 MB/s)
[INFO] 
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ tika-dl ---
[INFO] 
[INFO] --- maven-enforcer-plugin:3.0.0-M1:enforce (enforce) @ tika-dl ---
Downloading: https://oss.sonatype.org/content/repositories/snapshots/org/bytedeco/javacpp/1.4.2-SNAPSHOT/javacpp-1.4.2-20180705.150048-31.pom
Downloaded: https://oss.sonatype.org/content/repositories/snapshots/org/bytedeco/javacpp/1.4.2-SNAPSHOT/javacpp-1.4.2-20180705.150048-31.pom (0 B at 0 B/s)
Downloading: http://maven.restlet.org/org/nd4j/nd4j-backend-impls/1.0.0-SNAPSHOT/maven-metadata.xml
Downloading: https://oss.sonatype.org/content/repositories/snapshots/org/nd4j/nd4j-backend-impls/1.0.0-SNAPSHOT/maven-metadata.xml
Downloading: http://scala-tools.org/repo-releases/org/nd4j/nd4j-backend-impls/1.0.0-SNAPSHOT/maven-metadata.xml
Downloaded: https://oss.sonatype.org/content/repositories/snapshots/org/nd4j/nd4j-backend-impls/1.0.0-SNAPSHOT/maven-metadata.xml (826 B at 3.4 kB/s)
[WARNING] 
Dependency convergence error for net.java.dev.jna:jna:4.1.0 paths to dependency are:
+-org.apache.tika:tika-dl:2.0.0-SNAPSHOT
  +-org.apache.tika:tika-parsers:2.0.0-SNAPSHOT
    +-edu.ucar:netcdf4:4.5.5
      +-net.java.dev.jna:jna:4.1.0
and
+-org.apache.tika:tika-dl:2.0.0-SNAPSHOT
  +-org.deeplearning4j:deeplearning4j-nn:1.0.0-SNAPSHOT
    +-com.github.oshi:oshi-core:3.4.2
      +-net.java.dev.jna:jna-platform:4.3.0
        +-net.java.dev.jna:jna:4.3.0

[WARNING] 
Dependency convergence error for org.bytedeco:javacpp:1.4.2-SNAPSHOT paths to dependency are:
+-org.apache.tika:tika-dl:2.0.0-SNAPSHOT
  +-org.deeplearning4j:deeplearning4j-nn:1.0.0-SNAPSHOT
    +-org.nd4j:nd4j-native-api:1.0.0-SNAPSHOT
      +-org.bytedeco:javacpp:1.4.2-SNAPSHOT
and
+-org.apache.tika:tika-dl:2.0.0-SNAPSHOT
  +-org.bytedeco:javacpp:1.4.1

[WARNING] Rule 0: org.apache.maven.plugins.enforcer.DependencyConvergence failed with message:
Failed while enforcing releasability. See above detailed error message.
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 24.913 s
[INFO] Finished at: 2018-07-08T21:52:38-07:00
[INFO] Final Memory: 25M/414M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-enforcer-plugin:3.0.0-M1:enforce (enforce) on project tika-dl: Some Enforcer rules have failed. Look above for specific messages explaining why the rule failed. -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
nonas:tika-dl mattmann$ 
```

--- a/pom.xml
+++ b/pom.xml
@@ -177,7 +177,7 @@
         <javacpp.version>1.4.2-SNAPSHOT</javacpp.version>
         <javacpp-presets.version>1.4.2-SNAPSHOT</javacpp-presets.version>
         <javacv.version>1.4.2-SNAPSHOT</javacv.version>
-        <openblas.version>0.3.0</openblas.version>
+        <openblas.version>0.3.1</openblas.version>
         <mkl.version>2018.3</mkl.version>
         <mkl-dnn.version>0.14</mkl-dnn.version>
         <opencv.version>3.4.1</opencv.version>

